### ISSUE TITLE
ClipsToBounds Problem

### ISSUE THREAD

**Issue by gasparuff** [m1entus/MZFormSheetPresentationController#148]
I'm trying to move an Imageview from my presented formsheet to my tabbar with an animation like in this project here: https://github.com/pratik-123/FlyToCart

But as soon as the imageview gets moved towards the tab bar, it gets clipped once its outside the mzformsheet's bounds. In the view hierarchy I can see that a superview called UINavigationTransitionView has it's clipsToBounds property set to true, but I can't find any way of how to change this.

Can anyone help me out?

**Comment by m1entus** [comment]
https://github.com/m1entus/MZFormSheetPresentationController/blob/master/MZFormSheetPresentationController/MZFormSheetPresentationViewController.m#L320

**Comment by gasparuff** [comment]
Changing self.contentViewController.view.layer.masksToBounds didn't have any impact :-(

**Comment by gasparuff** [comment]
Interestingly it works when the MZFormSheetPresentationViewController is instantiated using a UIViewController instead of a UINavigationController

**Comment by gasparuff** [comment]
Ok I found the problem. It actually has nothing to do with your library, but with the UINavigationController. It creates 2 subviews - UILayoutContainerView and UINavigationTransitionView. The latter has clipsToBounds set to true, which is the reason why it didn't work. For now I have changed that value in the viewDidLoad() function of my presented viewcontroller:

` self.navigationController?.view.subviews.first?.clipsToBounds = false`

Kinda hacky but I don't see any other solution right now.
